openshift/templates/aries-mediator-service/aries-mediator-service-deploy.yaml

```diff
@@ -7,7 +7,31 @@ metadata:
     tags: node
     iconClass: icon-node
 objects:
-
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: ${NAME}-allow-ingress
+      labels:
+        name: ${NAME}-allow-ingress
+        app: ${APP_NAME}
+        app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
+    spec:
+      description: |
+        Allow traffic from routes into the ${ROLE} container(s).
+      ingress:
+        - from:
+            - namespaceSelector:
+                matchLabels:
+                  network.openshift.io/policy-group: ingress
+      podSelector:
+        matchLabels:
+          role: ${ROLE}
+          app: ${APP_NAME}
+          env: ${TAG_NAME}
+      policyTypes:
+        - Ingress
```

Comment:
The network policy should follow the ${NAME}${SUFFIX} naming pattern to support multiple instance within the same environment. 
Use https://github.com/bcgov/orgbook-configurations/blob/main/openshift/templates/agent/agent-deploy.yaml as the reference.

Reply:
Change completed